coordinator-common/src/test/java/org/apache/kafka/coordinator/common/runtime/EventAccumulatorTest.java

```diff
@@ -118,9 +118,9 @@ public void testAddFirst() {
         EventAccumulator<Integer, MockEvent> accumulator = new EventAccumulator<>();
 
         List<MockEvent> events = Arrays.asList(
```

Comment:
List.of.

Reply:
We do `Collections.reverse()` below which requires a mutable list.